scanner/scanner.go

```diff
@@ -714,6 +715,10 @@ func parseMulti(parser tagcommon.Info, setting MultiValueSetting, getMulti func(
 	default:
 		parts = []string{get(parser)}
 	}
+	// trim potential empty strings from delimited results
```

Comment:
i think it would make sense to do this after the 
```go
for i := range parts {
		parts[i] = strings.TrimSpace(parts[i])
}
```
below

Reply:
Aha, you are correct, thanks!